projects/bitcoin-core/Dockerfile

```
@@ -23,7 +23,7 @@ RUN apt-get update && apt-get install -y \
   automake autotools-dev bsdmainutils build-essential cmake curl g++-multilib libtool make \
   patch pkg-config python3 wget zip
 
-RUN git clone --depth=1 https://github.com/bitcoin/bitcoin.git bitcoin-core
+RUN git clone --depth=1 --branch cmake-staging https://github.com/hebasto/bitcoin.git bitcoin-core
```

```suggestion
RUN git clone --depth=1 --branch 240716-cmake https://github.com/hebasto/bitcoin.git bitcoin-core
```

I presume this will be the devel branch going forward?

---

Sure thing! Updated.